cypress/integration/person.js

```diff
@@ -1,6 +1,8 @@
 describe('Person Visualization Check', () => {
     beforeEach(() => {
         cy.get('[data-attr=menu-item-people]').click() // TODO: Adjust when releasing navigation-1775
+        cy.get('[data-attr=menu-item-persons].ant-menu-item-selected').should('be.visible')
```

Comment:
Context on these changes? Seem unrelated to the PR

Reply:
Demo data changed making changes on how navigation works needed. 

In this case, there are tabs on the page we start from so we need to wait for the new page to load first.

Ideally we should detangle e2e tests from demo data but that's a longer spiel.